composer.json

```diff
@@ -29,7 +29,9 @@
         "behat/web-api-extension": "~1.0@dev",
         "behat/symfony2-extension": "~2.0@dev",
         "phpbench/phpbench": "~0.10.0",
-        "phpbench/tabular": "@dev"
+        "phpbench/tabular": "@dev",
+
+        "guzzlehttp/streams": "^3.0.0"
```

Comment:
Hm, looks like that's a bug in the elasticsearch library? But well, looks like we don't have any other choice at the moment.